mscore/file.cpp

```diff
@@ -309,7 +313,9 @@ void MuseScore::openFiles(bool switchTab, bool singleFile)
              << tr("Bagpipe Music Writer Files (experimental)") + " (*.bmw *.bww)"
              << tr("Guitar Pro Files") + " (*.gtp *.gp3 *.gp4 *.gp5 *.gpx *.gp)"
              << tr("Power Tab Editor Files (experimental)") + " (*.ptb)"
-             << tr("MuseScore Backup Files") + " (*.mscz, *.mscx,)";
+           << tr("MuseScore Backup Files") + " (*.mscz, *.mscx,)"
+           << tr("MuseScore 4 Album Files") + " (*.msca *.mscaz)"
+           << tr("MuseScore 2 Album Files") + " (*.album)";
```

Comment:
formatting

Reply:
Also:
MuseScore **4** Album Files  -> MuseScore **3** Album Files

Reply:
yes, of course